test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointTest.java

```diff
@@ -764,4 +766,70 @@ public void persistedCorfuTableCheckpointTest() {
 
         }
     }
+
+    /**
+     * Test that validates that a DiskBacked CorfuTable can correctly serve snapshots
+     * after encountering a TrimmedException from the stream layer, even in the presence
+     * of EXPIRED snapshots in the MVOCache.
+     */
+    @Test
+    public void persistedCorfuTableCheckpointTrimTest() {
+        final String path = PARAMETERS.TEST_TEMP_DIR;
+        final int mvoCacheExpirySeconds = 3;
```

Comment:
Is there a reason why we cannot make this much smaller?

Reply:
Not particularly - Reduced from 3s to 1s.